intro-Linux/ch_filesystems.tex

```diff
@@ -3,36 +3,8 @@ \chapter{Filesystems}
 In this section, we briefly explain which different filesystems are available on
 the HPC infrastructure.
 
-See \url{http://hpc.ugent.be/userwiki/index.php/Tips:Filesystem_Information} and
-\url{http://hpc.ugent.be/userwiki/index.php/User:StorageDetails} for more
-information.
-
-\subsubsection{Home}
-
-Your own personal home directory: ``\$VSC\_HOME''
-
-You end up here when you log in (and when jobs start). This is meant for small
-configuration files, limited space available.
-
-\subsubsection{Data}
-
-Long-term personal storage (for large files, volumes): ``\$VSC\_DATA''
-
-\subsubsection{Scratch}
-
-Personal scratch (fast, but to be considered volatile) storage:
-``\$VSC\_SCRATCH''. ``\$VSC\_SCRATCH'' is just a short\-hand for the default
-scratch storage (``\$VSC\_SCRATCH\_DELCATTY'').
-
-\subsubsection{Local}
-
-When running jobs, you also have access to the local storage of a node (which
-you cannot access when logging in, as it is node-dependent).
-
-You access it through ``\$VSC\_SCRATCH\_NODE''.
-
-Inside jobs, a unique directory located in ``\$VSC\_SCRATCH\_NODE'' is made
-available via ``\$TMPDIR''.
+See \href{http://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-linux-gent.pdf#subsection.6.2.1}{Chapter 6.2.1 of the HPC manual}
```

Comment:
@redfast00 This link will easily go outdated if the section numbers change, so we should use a proper label here and mention something like `relevant section of the HPC manual` (or perhaps a bit more descriptive).

Also, rather than hard linking to `linux-gent`, we should take into account the `$OS` and `$SITE` variables somehow...

Reply:
I just wrote a macro to generate the url (taking into account the site and OS), but there are no labels in the exported intro-HPC pdf (and even if there were, jumping to them isn't supported by all readers). Do I link to the section and put the title in the description? I think that would be best of both worlds: users can always look it up (even if the order of chapters changes) and in a lot of the cases will be immediately directed to the correct chapter (but if it changes, they arrive at the wrong page). Another option is to just give the pdf link without setting a section/chapter. What do you want me to do?

Reply:
This PR is blocked by #223 for the links to the manuals.